v21.1/backup.md

```diff
@@ -69,6 +79,11 @@ The `BACKUP` statement cannot be used within a [transaction](transactions.html).
 CockroachDB uses the URL provided to construct a secure API call to the service you specify. The URL structure depends on the type of file storage you are using. For more information, see the following:
 
 - [Use Cloud Storage for Bulk Operations](use-cloud-storage-for-bulk-operations.html)
+
```

Comment:
~What is HTTP storage?  Like, POSTing a file to a random HTTP URL?  I think this might be clearer as something like "Neither BACKUP nor RESTORE support backing up to HTTP storage" (whatever that is :-} )  If we have a link to something describing what HTTP storage is, that would be great